assets/test/SignPanel.test.ts

```diff
@@ -599,3 +599,49 @@ test('does not save changes to backend until Apply is pressed', () => {
     },
   ]);
 });
+
+test("shows Apply button but not expiration form for signs with no 'Auto' mode", () => {
+  const now = new Date('2019-01-15T20:15:00Z').valueOf();
+  const fresh = new Date(now + 5000).toLocaleString();
+  const currentTime = now + 2000;
+
+  const setConfigHistory: SignConfig[] = [];
+
+  render(
+    React.createElement(SignPanel, {
+      alerts: {},
+      signId: 'signId',
+      modes: { auto: false, custom: true, headway: false, off: true },
+      signContent: signContentWithExpirations(fresh, fresh),
+      currentTime: currentTime,
+      line: 'Silver',
+      setConfig: (signConfig: SignConfig) => {
+        setConfigHistory.push(signConfig);
+      },
+      signConfig: { mode: 'auto' },
```

Comment:
Would this be possible in reality, given `auto: false` above?

Reply:
Good catch. I'll change the initial mode to `'off'` like it usually is in real life.